### PR TITLE
Jetpack App (Basics): Consolidate empty view layouts

### PR DESCRIPTION
**Pull request by zwarm**
Parent #14302 

This PR consolidates the layouts for the Jetpack Empty Views. 
- Deleted `fragment_login_no_sites_empty_view.xml` and `fragment_login_site_check_error.xml`
- Created `jetpack_login_empty_view` as the main empty view
- Created separate User and Button container "include layouts"
- Updated `LoginNoSiteFragment` and `LoginSiteCheckErrorFragment` to use the `jetpack_login_empty_view`. 

**To test:**
**Scenario 1: No Jetpack Sites Empty View**
- Download this branch and build a Jetpack App variant
- Tap the 'Login with WordPress' button
- Enter WordPress.com credentials for an account that is not linked to any Jetpack sites
- After logging in you should be brought to the No Jetpack Sites Found empty view

**Scenario 2: Not a Jetpack Site Empty View**
- Download this branch and build a Jetpack App variant
- Tap the 'Enter your existing address' button
- Enter `http://do.wpmt.co/jp-disconnected/` or another site that has Jetpack installed but not connected
- You should be brought to an empty view with the message "The website {site} is not a Jetpack site"

## Regression Notes
1. Potential unintended areas of impact 🟢 
2. What I did to test those areas of impact (or what existing automated tests I relied on) 🟢 
- Manual testing + ran existing tests
3. What automated tests I added (or what prevented me from doing so) 🟢 

PR submission checklist:

- [X] I have completed the Regression Notes.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/login/jetpack/LoginNoSitesFragment.kt [hunk 1]
```diff
@@ -86,27 +92,31 @@ class LoginNoSitesFragment : Fragment(R.layout.fragment_login_no_sites_empty_vie
...
+    private fun JetpackLoginEmptyViewBinding.loadGravatar(avatarUrl: String) {
         val url = meGravatarLoader.constructGravatarUrl(avatarUrl)
-        AvatarHelper.loadAvatarFromUrl(this@LoginNoSitesFragment, url, imageAvatar, object : AvatarRequestListener {
-            override fun onRequestFinished() {
-                // no op
-            }
-        })
+        AvatarHelper.loadAvatarFromUrl(
+                this@LoginNoSitesFragment,
+                url,
```
Comment (ParaskP7) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/login/jetpack/LoginNoSitesFragment.kt]: Minor (🔍): Consider inlining the `url` here.

File: WordPress/src/main/res/layout/jetpack_login_empty_view.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ParaskP7) [inline WordPress/src/main/res/layout/jetpack_login_empty_view.xml:1]: Minor (🔍): Consider reformatting this xml file.

PS: Same applies for the `jetpack_login_include_user_container.xml` file.

**Review by ParaskP7 — APPROVED** [review]
👋 @zwarm !

I have reviewed and tested this PR as per the instructions, everything works as expected, thanks so much for removing this duplication, such a good job! 🌟 🌟 

I have left a couple of minor (🔍) comments for you to consider. I am going to approve this PR anyway, since none is blocking. I am **NOT** going to merge this PR yet to give you some time to apply any of my suggestions. However, feel free to ignore them and merge the PR yourself.

File: WordPress/src/main/res/layout/jetpack_login_empty_view.xml [hunk 3]
```diff
@@ -2,78 +2,78 @@
...
-        app:layout_constraintBottom_toTopOf="@+id/login_empty_view_illustration"
-        android:visibility="invisible">
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/user_cardView"
+            style="@style/Widget.EmptyView.CardView.BottomBar"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            app:cardCornerRadius="0dp"
+            app:cardElevation="0dp"
+            tools:visibility="visible"
```
Comment (ParaskP7) [inline WordPress/src/main/res/layout/jetpack_login_empty_view.xml]: 👋 @zwarm !

If I reformat this file the `android:xyz` attributes will go on top of the `tools:xyz`, but this didn't happen here, I wonder what is the problem?

Comment (zwarm) [reply WordPress/src/main/res/layout/jetpack_login_empty_view.xml]: @ParaskP7 - Your guess is as good as mine. Constraint layouts seem to be a an AS reformat nightmare. I'm leaving it as it is.

Comment (ParaskP7) [reply WordPress/src/main/res/layout/jetpack_login_empty_view.xml]: @zwarm in my case it was doing at least what I expected with that. Do you want a quick 5' mins pairing to see what is happening on your IDE?

PS: The other file you reformatted worked as expected.

Comment (zwarm) [reply WordPress/src/main/res/layout/jetpack_login_empty_view.xml]: @ParaskP7 - Sure, we can compare Editor -> Code Style -> XML in the preferences settings

**Pull request merged**